packages/plugin-pack/sources/packUtils.ts

```diff
@@ -117,6 +117,8 @@ export async function genPackStream(workspace: Workspace, files?: Array<Portable
         pack.entry({...opts, mode, type: `file`}, content, cb);
       } else if (stat.isSymbolicLink()) {
         pack.entry({...opts, mode, type: `symlink`, linkname: await xfs.readlinkPromise(source)}, cb);
+      } else {
+        cb(null);
```

Comment:
Shouldn't it reject? 🤔

Reply:
I'm not sure what we should do here

Reply:
I'd go with rejecting with a message like `Unsupported file type [...] for [...]`